resources/2023-10-recipe-details.md

```diff
@@ -0,0 +1,231 @@
+# Title
+
+* **Status**: In Progress
+* **Author**: Yetkin Timocin (@ytimocin)
+
+## Overview
+
+Recipes is a feature that is offered by Radius to make the deployment of resources easier for its users. To learn more about recipes please visit <https://edge.radapp.dev/guides/recipes/overview/>. If a resource is deployed by a recipe, we would like to know this in the resource and data model level.
+
+## Terms and definitions
+
+| Term | Definition |
+|---|---|
+| Recipe | Recipes enable a separation of concerns between infrastructure operators and developers by automating infrastructure deployment. |
+| Template Kind | The format of the template provided by the recipe. Allowed values: bicep, terraform. |
+| Template Path | The path to the template provided by the recipe. |
+| Template Version | The version of the template to deploy. Only applies to Terraform recipes. |
+
+## Objectives
+
+> **Issue Reference:**
+
+<https://github.com/radius-project/radius/issues/6440>
+
+### Goals
+
+* The goal is to add the recipe information to the resource and the data model level so that when the resource details is requested, recipe information will also be available.
+
+### Non goals
+
+* Everything other than adding the recipe information to the resource and data model level.
+* No changes to any of the existing APIs. The APIs will automatically pick up the changes.
+
+### User scenarios (optional)
```

Comment:
Should this also include the application graph API?

Reply:
I think we have another design document for the Application Graph.

Reply:
https://github.com/radius-project/design-notes/blob/main/resources/2023-10-app-graph.md